Paco-Server/ear/default/web/js/controllers.js

```diff
@@ -1,708 +1,724 @@
-          preserveScope : true,
-          locals : {
-            experiment : experiment,
-            anonymous : $scope.anon,
-          },
-          clickOutsideToClose : true,
-          controller : 'ReportCtrl'
-        });
-        promise.then(function(result) {
-          $scope.reportData = result.data;
-          $scope.reportType = result.type;
-        });
-      };
+      $scope.status = 'Requesting response data';
+    };
 
-      $scope.$watch('statsDate', function(newVal, oldVal) {
-        if (oldVal !== newVal) {
-          dataService.updateParticipantDateStats($scope.experimentId, $scope.statsDate, $scope.stats);
-        }
-        ;
+    $scope.showReportOptions = function (experiment) {
+      var promise = $mdDialog.show({
+        templateUrl: 'partials/report.html',
+        preserveScope: true,
+        locals: {
+          experiment: experiment,
+          anonymous: $scope.anon,
+        },
+        clickOutsideToClose: true,
+        controller: 'ReportCtrl'
+      });
+      promise.then(function (result) {
+        $scope.reportData = result.data;
+        $scope.reportType = result.type;
       });
+    };
 
-      $scope.loadStats = function() {
-        $scope.loading = true;
-        $scope.stats = null;
-        $scope.currentView = 'stats';
+    $scope.$watch('statsDate', function (newVal, oldVal) {
+      if (oldVal !== newVal) {
+        dataService.updateParticipantDateStats($scope.experimentId, $scope.statsDate, $scope.stats);
+      }
+      ;
+    });
 
-        dataService.getParticipantStats($scope.experimentId, $scope.statsDate, $scope.restrict, $scope.showGroup).then(
-            function(result) {
+    $scope.loadStats = function () {
+      $scope.loading = true;
+      $scope.stats = null;
+      $scope.currentView = 'stats';
 
-              if (result.data) {
-                $scope.stats = result.data;
-                $scope.loading = false;
-              }
-            }, function(result) {
+      dataService.getParticipantStats($scope.experimentId, $scope.statsDate, $scope.restrict, $scope.showGroup).then(
+          function (result) {
+
+            if (result.data) {
+              $scope.stats = result.data;
               $scope.loading = false;
-              $scope.error = {
-                code : result.status,
-                message : result.statusText
-              };
-            });
+            }
+          }, function (result) {
+            $scope.loading = false;
+            $scope.error = {
+              code: result.status,
+              message: result.statusText
+            };
+          });
 
-        $scope.status = 'Sending stats request';
-      }
+      $scope.status = 'Sending stats request';
+    }
 
-      $scope.isPhotoData = function(data) {
-        return (typeof (data) === 'string' && (data.indexOf($scope.photoMarker) === 0 || data.indexOf("iVBORw0K") === 0));
-      }
+    $scope.isPhotoData = function (data) {
```

Comment:
Please fix formatting errors that add a space in between function and the paren.